syntax = "proto3";
package chat.proto;

message MessagePush {
    string topic = 1;
    string content = 2;
    string from = 3;
}
